tardis/visualization/tools/sdec_plot.py

```diff
@@ -1105,6 +1106,8 @@ def generate_plot_mpl(
         distance : astropy.Quantity or None, optional
             Distance used to calculate flux instead of luminosity in the plot.
             It should have a length unit like m, Mpc, etc. Default value is None
+        observed_spectrum : tuple or list of two astropy quantities, optional
+            Option to plot the observed spectrum.
```

Comment:
Explain what will be the list items and what are allowed quantities. Read `packet_wvl_range` and `distance` parameter description for inspiration